common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
Is there an alternative to using JQuery UI for this? It is a 230 KB library that we have tried to excise as much as possible for performance reasons. Introducing it here will mean that all of our pages will end up loading it again.